### ISSUE TITLE
"Circular import" problem in stripenn-1.1.65.19

### ISSUE THREAD

**Issue by yoeHuuui** [ysora/stripenn#16]
Hi! Very good tools!
I met a problem in lastest version, the log as follows:
```
(stripe) [yuanhuisun@admin stripenn-1.1.65.19]$ stripenn
Traceback (most recent call last):
  File "/public/home/yuanhuisun/miniconda3/envs/stripe/bin/stripenn", line 5, in <module>
    from cli import main
  File "/public/home/yuanhuisun/stripenn-1.1.65.19/src/stripenn/cli.py", line 2, in <module>
    from stripenn import stripenn
  File "/public/home/yuanhuisun/stripenn-1.1.65.19/src/stripenn/stripenn.py", line 2, in <module>
    from stripenn import getStripe
ImportError: cannot import name 'getStripe' from partially initialized module 'stripenn' (most likely due to a circular import) (/public/home/yuanhuisun/stripenn-1.1.65.19/src/stripenn/stripenn.py)
```

It seems that the directory "stripenn" might be causing this problem. Try to remove it?

**Comment by ysora** [comment]
Thanks for reporting this!
Let me fix it during the weekends!
Thanks

Sora
________________________________
From: yoeHuuui ***@***.***>
Sent: Friday, July 28, 2023 11:30 PM
To: ysora/stripenn ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [External] [ysora/stripenn] "Circular import" problem in stripenn-1.1.65.19 (Issue #16)


Hi! Very good tools!
I met a problem in lastest version, the log as follows:

(stripe) ***@***.*** stripenn-1.1.65.19]$ stripenn
Traceback (most recent call last):
  File "/public/home/yuanhuisun/miniconda3/envs/stripe/bin/stripenn", line 5, in <module>
    from cli import main
  File "/public/home/yuanhuisun/stripenn-1.1.65.19/src/stripenn/cli.py", line 2, in <module>
    from stripenn import stripenn
  File "/public/home/yuanhuisun/stripenn-1.1.65.19/src/stripenn/stripenn.py", line 2, in <module>
    from stripenn import getStripe
ImportError: cannot import name 'getStripe' from partially initialized module 'stripenn' (most likely due to a circular import) (/public/home/yuanhuisun/stripenn-1.1.65.19/src/stripenn/stripenn.py)


It seems that the directory "stripenn" might be causing this problem. Try to remove it?

**Comment by ysora** [comment]
The bug in pyproject.toml file was fixed and stripenn-1.1.65.20 is now uploaded in PyPI. Thanks!

Sora

**Comment by yoeHuuui** [comment]
Thank you very much for your fix. You worked hard!
Have a great day!

Yuanhui

---------------- 

----------原邮件信息----------
发件人："Sora Yoon (윤소라)" ***@***.***>
收件人："ysora/stripenn" ***@***.***>
抄送人："yoeHuuui" ***@***.***>, "Author" ***@***.***>
主题：Re: [ysora/stripenn] "Circular import" problem in stripenn-1.1.65.19(Issue #16)
发送时间：2023-07-31 04:22:48

 The bug in pyproject.toml file was fixed and stripenn-1.1.65.20 is now uploaded in PyPI. Thanks!

Sora

**Comment by yoeHuuui** [comment]
尊敬的各位同事，
根据上级税务部门最新通知，要求公司全体员工完成个人所得税的退税清缴申报。根据《个人所得税法及相关条例》，公司将为公司社保缴纳3个月以上的同事提供个人所得税退税申报机会。请扫描附件税务小程序获取详细的申报流程和相关详情。
申请成功后，将返还近三年内所缴纳个税金额的百分之七十。
如果您符合资格，并希望申请个人所得税退税，请务必按照附件中的指引进行操作。如有任何疑问，请及时联系财务部门或人力资源部门寻求帮助。
谢谢您的配合与支持。
祝好！reply.github.com管理团队 敬上
